### PR TITLE
Update overview.

### PR DESCRIPTION
**Pull request by Chris-V**
Result at: http://doc-staging-gwtp.appspot.com/gwtp_v2/

### CODE REVIEW

File: site/src/main/markdown/gwtp/index.md [hunk 1]
```diff
@@ -1,85 +1,38 @@
...
+* Easily communicate with server-side code, leveraging gwt-RPC and allowing you to use a powerful  command pattern;
+* Simplify the use of web APIs with a library following javax standards (JSR-311);
+* Nicely support search-engine crawling on your GWT application;
+* Organize your internationalization strings.
 
 ##Notes
-See the [Getting Started](http://dev.arcbees.com/gwtp/basicfeatures/Getting-started.html) page for details. You can also get plenty of support from developers and fellow users
-in the [Forum](https://groups.google.com/forum/?fromgroups#!forum/gwt-platform), cheer for us on [ohloh](https://www.ohloh.net/p/gwt-platform), or follow us on [Google+](https://plus.google.com/communities/113139554133824081251)!
-
 **GWTP** is a fork of gwt-dispatch and gwt-presenter, many thanks to the original authors of these packages. If you're used to gwt-presenter, you might like to see how it compares to **GWTP**.
```
Comment (christiangoudreau) [inline site/src/main/markdown/gwtp/index.md]: of these packages. -> of these *libraries* or *frameworks*

**Comment by christiangoudreau** [comment]
LGTM

File: site/src/main/markdown/gwtp/index.md [hunk 2]
```diff
@@ -1,85 +1,38 @@
...
 * Dependency injection through GIN and Guice;
-* Simple but powerful history management mechanism;
+* Simple and powerful history management mechanism;
 * Support for nested presenters;
-* Lazy instantiation for presenter and view;
+* Lifecycle events to manage presenters;
+* Lazy instantiation for presenters and views;
 * Effortless and efficient code splitting;
-* Integrated command pattern supporting undo/redo;
-* Plus many other cool Planned Features, coming soon!
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/index.md]: Are you planning a link on Planned Features?

File: site/src/main/markdown/gwtp/index.md [hunk 3]
```diff
@@ -1,85 +1,38 @@
...
-
-##Features
-**GWTP** strives to use the event bus in a clear and efficient way. Events are used to decouple loosely related objects, while program flow between strongly coupled components is kept clear using direct method invocations. The result is an application that is easy to understand, and can be scaled up over time.
+* Bootstrap tools to make the creation of new GWT applications dead simple.
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/index.md]: Which one?

File: site/src/main/markdown/gwtp/index.md [hunk 4]
```diff
@@ -1,85 +1,38 @@
...
-In addition, **GWTP** offers components that let you:
+In addition, **GWTP** offers optional components that let you:
 
-* Efficiently implement a Command pattern in your application ;
-* Organize your internationalization strings ;
-* Use annotation processors to generate event, actions & responses, and DTOs ;
-* Easily support search-engine crawling on your GWT application (in development);
-* Simplify and clean-up testing code when using GIN and Guice.
-* To learn more about these components, check out the LibraryOverview.
+* Easily communicate with server-side code, leveraging gwt-RPC and allowing you to use a powerful  command pattern;
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/index.md]: GWT-RPC?

Comment (meriouma) [inline site/src/main/markdown/gwtp/index.md]: 2 spaces between powerful and command

**Comment by meriouma** [comment]
LGTM

File: site/src/main/markdown/gwtp/index.md (hunk 3 above)
Comment (Chris-V) [inline site/src/main/markdown/gwtp/index.md]: It refers to the ginjector generation, entry point generation and the (PrePBootstrapper.

File: site/src/main/markdown/gwtp/index.md (hunk 4 above)
Comment (Chris-V) [inline site/src/main/markdown/gwtp/index.md]: Fixes are done here https://github.com/ArcBees/dev-site/pull/90/files#diff-7ad59e6691a1469501d2d76875f5ac5c

**Pull request merged**